src/main/java/com/softserve/app/controller/UserController.java

```diff
@@ -3,31 +3,82 @@
 import com.fasterxml.jackson.annotation.JsonView;
 import com.softserve.app.constant.SportHubConstant;
 import com.softserve.app.dto.UserDTO;
+import com.softserve.app.exception.SportHubException;
+import com.softserve.app.models.ResetPasswordRequest;
 import com.softserve.app.models.User;
 import com.softserve.app.models.View;
-import com.softserve.app.repository.UserRepository;
-import com.softserve.app.models.ResetPasswordRequest;
 import com.softserve.app.service.ResetService;
-import com.softserve.app.service.userService.UserService;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiParam;
+import com.softserve.app.service.UserService.UserService;
 import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.util.List;
 import java.util.Objects;
 
 
 @RestController
+@CrossOrigin(origins = "http://localhost:4200")
 @AllArgsConstructor
+@RequestMapping("/user/{user_id}")
```

Comment:
RequestMapping("/user/")

Reply:
because in the case of findAll, where you don't need id, you will fill it from the FE side.

Reply:
in your case